targets/fomu/Makefile.mk

```diff
@@ -0,0 +1,79 @@
+# fomu targets
+
+ifneq ($(PLATFORM),fomu)
+	$(error "Platform should be ice40-up5k-uwg30 or ice40-up5k-sg48 when using this file!?")
+endif
+
+PPATH := $(PYTHONPATH)
+export PYTHONPATH=$(PPATH):$(TARGET_BUILD_DIR)/../../third_party/valentyusb/
+
+# Settings
+DEFAULT_TARGET = base
+TARGET ?= $(DEFAULT_TARGET)
+BAUD ?= 115200
+
+MAKE_LITEX_EXTRA_CMDLINE += "--no-compile-software"
+
+FOBOOT_BIN = $(TARGET_BUILD_DIR)/../../rtl/foboot.bin
+FOBOOT_DIR = $(TARGET_BUILD_DIR)/../../third_party/foboot/sw
+
+# Image
+image-flash-$(PLATFORM):
+	iceprog $(IMAGE_FILE)
```

Comment:
`iceprog` isn't use with Fomu -- you want `dfu-util`

Reply:
At this point the `base` target for Fomu builds the whole bootloader image (gateware + foboot as bios). I think that loading it must be done using `iceprog` and a direct flash connection. Or is it possible to update the bootloader using `dfu-util`?

Reply:
It is possible to update the bootloader via dfu-util